package.json

```diff
@@ -30,13 +30,16 @@
     "npm": "please_use_yarn_instead"
   },
   "scripts": {
-    "build": "yarn workspace @graphiql/react run build && yarn tsc --clean && yarn tsc",
+    "build": "yarn build:clean && yarn build:graphiql-react && yarn build:packages && yarn build:graphiql",
```

Comment:
I had to make sure the builds are executed in correct order since we use Vite for `@graphiql/react`:
- First build all packages
- Then build `@graphiql/react`
- Then build `graphiql`